scripts/tasks/codepen-examples-transform.js

```diff
@@ -0,0 +1,112 @@
+/*
+This Transform modifies Fabric website code examples into a format that will allow them to be rendered on Codepen, as part of the "Export to Codepen" feature.
+There are two types of supported example templates:
+1)
+[imports]
+[variable export named declaration with example code inside]
+and
+2)
+[imports]
+[class export named declaration with example code inside. Having multiple methods and keeping track of state is supported]
+Currently, examples which are scattered across multiple files are NOT supported.
+*/
+const babylon = require('babylon');
+const recast = require('recast');
+
+function parseRaw(code) {
+  const parse = source =>
+    babylon.parse(source, {
+      sourceType: 'module',
+      plugins: ['jsx', 'typescript', 'classProperties']
+    });
+  return recast.parse(code, { parser: { parse } }).program.body;
+}
+
+function transform(file, api) {
+  const parse = source =>
+    babylon.parse(source, {
+      sourceType: 'module',
+      plugins: ['jsx', 'typescript', 'classProperties']
+    });
+
+  const j = api.jscodeshift;
+
+  let source = j(recast.parse(file.source, { parser: { parse } }));
+
+  //remove react import
+  source = source
+    .find(j.ImportDeclaration, node => node.source.value == 'react')
+    .remove()
+    .toSource();
+  source = j(recast.parse(source, { parser: { parse } }));
+
+  //remove css imports
+  source = source
+    .find(j.ImportDeclaration, node => node.source.value.endsWith('css'))
```

Comment:
I think we should use something like '.scss' & '.css' as tests - if we happened to have named an import something that ends with css, it would get caught and get removed here even if it isn't an extension.